src/external-site/_vars-external-site.html

```diff
@@ -6,7 +6,7 @@
    ========================================================================== */
 
     // Find the external-urls in your local cfgov-refesh repo.
-    find . -name '*.html' ! -path './src/*' ! -path './node_modules/*' ! -path './_tests/*' \
+    find . -name '*.html' ! -path './*' ! -path '../node_modules/*' ! -path '../test/*' \
```

Comment:
Cool. Just noticing this is wrapped in a code comment. So you'd copy and paste this into a script in root and run it? Is that right? How often does it need to be run and what's the relationship (if any) to `./links.sh`?